pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/MessageDeduplication.java

```diff
@@ -337,12 +338,23 @@ public MessageDupStatus isDuplicate(PublishContext publishContext, ByteBuf heade
             publishContext.setOriginalHighestSequenceId(highestSequenceId);
             headersAndPayload.readerIndex(readerIndex);
         }
-
+        long chunkID = 0;
```

Comment:
If the message is not a chunked message, the chunk ID will be `0`, right? Maybe setting it to `-1` is better?

Reply:
We need to check the sequence ID of the first chunk, just as a common message.  So it is fine to set it to `0`, right?